src/prefect/client/orchestration.py

```diff
@@ -2334,6 +2334,21 @@ async def read_artifacts(
         response = await self._client.post("/experimental/artifacts/filter", json=body)
         return pydantic.parse_obj_as(List[Artifact], response.json())
 
+    async def delete_artifact(self, artifact_id: str) -> None:
```

Comment:
should this be UUID?
```suggestion
    async def delete_artifact(self, artifact_id: UUID) -> None:
```